tests/core/test_cycles.py

```diff
@@ -59,7 +58,7 @@ class variable5(Variable):
     definition_period = MONTH
 
     def formula(person, period):
-        variable6 = person('variable6', period.last_month, max_nb_cycles = 0)
```

Comment:
Some comments above feel a little outdated. For instance does "but without explicit cycle allowed" still make sense now that it's not the formula writer responsibility to allow a cycle or not?